fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/HudRenderCallback.java

```diff
@@ -19,11 +19,12 @@
 import net.minecraft.client.gui.DrawContext;
 import net.minecraft.client.render.RenderTickCounter;
 
+import net.fabricmc.fabric.api.client.rendering.v1.hud.HudElementRegistry;
 import net.fabricmc.fabric.api.event.Event;
 import net.fabricmc.fabric.api.event.EventFactory;
 
 /**
- * @deprecated Use {@link HudLayerRegistrationCallback} instead. For common use cases, see {@link LayeredDrawerWrapper}.
+ * @deprecated Use {@link HudElementRegistry} instead. For common use cases, see {@link HudElementRegistry}.
```

Comment:
```suggestion
 * @deprecated Use {@link HudElementRegistry} instead.
```
No need to link twice to the same class